### PR TITLE
Use daily-react repo and new components

### PR DESCRIPTION
**Pull request by jayne-mast**
This uses the new `daily-react` repo in favour of `daily-react-hooks`.

With the new `daily-react` repo, we can now also use `DailyVideo` and `DailyAudio`, which removes a whole bunch of code.

### CODE REVIEW

File: src/components/Tile/Tile.js [hunk 1]
```diff
@@ -1,39 +1,21 @@
...
-        }
-      }
-    }, [audioTrack]);
   }
 
   return (
     <div className={containerCssClasses}>
-      <TileVideo id={id} isScreenShare={isScreenShare} />
-      {!isLocal && audioTrack && <audio autoPlay playsInline ref={audioElement} />}
+      <DailyVideo automirror sessionId={id} type={isScreenShare ? 'screenVideo' : 'video'} />
```
Comment (lazeratops) [inline src/components/Tile/Tile.js:17]: To make sure I understand: Does `DailyVideo` contain both audio and video, and does it already not play audio for the local participant by default? Eg. will this work as expected if the video is disabled but the audio is not?

**Review by lazeratops — APPROVED** [review]
LGTM with just one question for my own understanding.

File: src/components/Tile/Tile.js (hunk 1 above)
Comment (nienkedekker) [reply src/components/Tile/Tile.js:17]: If I understand correctly, DailyAudio is now loaded in `Call.js`, so if you disable video, it shouldn't affect the audio. `DailyAudio` takes care of _everyone_'s audio, and isn't tied to a tile or a participant any longer.

Comment (jayne-mast) [reply src/components/Tile/Tile.js:17]: Yes, what @nienkedekker said is correct. But if you want more control over each audio track, you can use `DailyAudioTrack` instead.

**Pull request merged**